src/docs/asciidoc/resources/migration_guide_to_4.6.adoc

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2024 RTE (http://www.rte-france.com)
+// See AUTHORS.txt
+// This document is subject to the terms of the Creative Commons Attribution 4.0 International license.
+// If a copy of the license was not distributed with this
+// file, You can obtain one at https://creativecommons.org/licenses/by/4.0/.
+// SPDX-License-Identifier: CC-BY-4.0
+
+= Migration Guide from release 4.5.X to release 4.6.0
+
+== Deprecated `toNotify` field
```

Comment:
The migration guide has to be referenced in src/docs/asciidoc/resources/index.adoc